City-of-Helsinki/drupal-module-helfi-tpr
# UHF-2837: Fixed service channel translations

To test this:

- Run `drush updb`
- Re-run errand service migration: `drush mim tpr_errand_service --update`.
- Re-run service channel migration: `drush mim tpr_service_channel --update` 
- Run `drush php-eval "foreach (\Drupal::state()->get('published_service_channels') as \$id) { array_map(function(\$langcode) use (\$id) { \Drupal\helfi_tpr\Entity\Channel::load(\$id)?->getTranslation(\$langcode)?->setPublished()->save(); }, ['fi', 'en', 'sv']); }"`

Make sure translations work and you see all available service channels. For example `/tpr-errand-service/4225` should have 5 service channels.
